pytition/pytition/settings/base.py

```diff
@@ -113,6 +113,9 @@
     # this enable mailer by default in django.send_email
     EMAIL_BACKEND = "mailer.backend.DbBackend"
 
+# set it to True if you use the 'mailer' backend, and a external Crontab has been set
```

Comment:
maybe also say in the comment that "using the mailer wihout uwsgi *and* without cron jobs is not supported and won't work"?
Maybe be very clear like: 
```
Only supported configurations:
- no mailer backend, emails are sent synchronously with no retry if sending fails
- mailer backend used with uwsgi without cron jobs
- mailer backend used without uwsgi with cron jobs
```